cmd/sidecar_mounter/main.go

```diff
@@ -189,5 +192,12 @@ func prepareMountConfig(sp string) (*sidecarmounter.MountConfig, error) {
 		return nil, fmt.Errorf("failed to fetch bucket name from CSI driver")
 	}
 
+	for _, opt := range mc.Options {
+		if strings.Contains(opt, "experimental-local-file-cache") {
+
+			mc.TempDir = filepath.Join(sidecarspec.CacheVolumeMountPath, "gcsfuse-tmp")
```

Comment:
One concern: does this work when there are multiple gcsfuse instances?

The previous design is that, for each gcsfuse process, it has a different temp dir. But with this change, I believe all the gcsfuse processes will share the same temp dir. I think this may introduce potential issues.

Reply:
Resolving as discussed offline. This feature no longer necessitates driver changes.